app/src/auto_validator/core/utils/childhotkey.py

```diff
@@ -34,78 +34,66 @@ def create_new_child_hotkey(
+            wallet_path=self.parent_wallet_path,
+            network=network,
+            netuid=netuid,
+            all_netuids=False,
+            children=[child_wallet.hotkey.ss58_address],
+            proportions=[proportion],
+            quiet=True,
+            verbose=False,
+            wait_for_finalization=True,
+            wait_for_inclusion=True,
+        )
+        sys.stdin = sys.__stdin__
         Hotkey.objects.create(hotkey=child_wallet.hotkey.ss58_address)
         return child_wallet.hotkey.ss58_address
 
     def get_child_hotkeys(self, network: str, netuid: int):
-        try:
-            cli_manager = CLIManager()
-            result = cli_manager.stake_get_children(
-                wallet_name=self.parent_wallet_name,
-                wallet_hotkey=self.parent_hotkey_name,
-                wallet_path=self.parent_wallet_path,
-                network=network,
-                netuid=netuid,
-                all_netuids=False,
-                quiet=True,
-                verbose=False,
-            )
-        except Exception as e:
-            self.logger.exception(f"Error while getting children hotkeys: {e}")
-            return None
+        cli_manager = CLIManager()
+        result = cli_manager.stake_get_children(
+            wallet_name=self.parent_wallet_name,
+            wallet_hotkey=self.parent_hotkey_name,
+            wallet_path=self.parent_wallet_path,
+            network=network,
+            netuid=netuid,
+            all_netuids=False,
+            quiet=True,
+            verbose=False,
+        )
         return result
 
     def revoke_child_hotkeys(
         self,
         network: str,
         netuid: int,
     ) -> bool:
-        try:
-            cli_manager = CLIManager()
-            sys.stdin = StringIO("y\n")
-            cli_manager.stake_revoke_children(
-                wallet_name=self.parent_wallet_name,
-                wallet_hotkey=self.parent_hotkey_name,
-                wallet_path=self.parent_wallet_path,
-                network=network,
-                netuid=netuid,
-                all_netuids=False,
-                quiet=True,
-                verbose=False,
-                wait_for_finalization=True,
-                wait_for_inclusion=True,
-            )
-            sys.stdin = sys.__stdin__
-        except Exception as e:
-            self.logger.exception(f"Error while revoking children hotkeys: {e}")
-            return False
+        cli_manager = CLIManager()
+        sys.stdin = StringIO("y\n")
+        cli_manager.stake_revoke_children(
+            wallet_name=self.parent_wallet_name,
+            wallet_hotkey=self.parent_hotkey_name,
+            wallet_path=self.parent_wallet_path,
+            network=network,
+            netuid=netuid,
+            all_netuids=False,
+            quiet=True,
+            verbose=False,
+            wait_for_finalization=True,
+            wait_for_inclusion=True,
+        )
+        sys.stdin = sys.__stdin__
```

Comment:
that sys.stdin thing should be in the CLIManager class, not in every single place where you are calling it